docs/docs/1.4.x/deployments/stand-alone.md

```diff
@@ -12,9 +14,32 @@ This mode implies that we can deploy Kuma and its data plane proxies in a standa
 <img src="/images/docs/0.6.0/flat-diagram.png" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
-Although standalone mode can support complex multi-zone or hybrid deployments (Kubernetes + VMs) as long as the networking requirements are satisfied, typically in most use cases our connectivity cannot be flattened out across multiple zones. Therefore standalone mode is usually a great choice within the context of one zone (ie: within one Kubernetes cluster or one AWS VPC).
+Standalone mode is usually a great choice within the context of one zone (ie: within one Kubernetes cluster or one AWS VPC).
+
+## Limitations
+
+* All dataplane-proxies need to be able to communicate with every other dataplane proxy.
+* A standalone deployment cannot mix Universal and Kubernetes workloads.
+* A deployment can connect to only one Kubernetes cluster at once.
+
+If these limitations are problematic you should look at [Multi-zone deployments](../multi-zone).
+
+## Failure modes
+
+#### Control plane offline
+
+* New data-planes won't be able to join the mesh.
+* Data-plane proxy configuration will not be updated.
+* Communication between data-planes will still work.
+* Cross zone communication will still work.
```

Comment:
cross zone communication in standalone mode?

Reply:
oops :)